pkgs/tools/security/collision/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, fetchFromGitHub
+, crystal
+, shards
+, libadwaita
+, openssl
+, wrapGAppsHook
+, pkg-config
+}:
+
+crystal.buildCrystalPackage rec {
+  pname = "icr";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "GeopJr";
+    repo = "Collision";
+    rev = "v${version}";
+    sha256 = "sha256-tzvQF4Gn0ppbst/fV4BGZuXEe7SXJBWcN6uIOXB2huc=";
+  };
+
+  shardsFile = ./shards.nix;
+
+  crystalBinaries.collision.src = "src/collision.cr";
+
+  nativeBuildInputs = [ wrapGAppsHook pkg-config ];
```

Comment:
Could you make the list one dependency per line? It will make diffs nicer to read and `git blame` easier to use.